Police-Data-Accessibility-Project/data-sources-app-v2
# Enhanced `/agencies` `GET` methods and tests to include linked data sources

### Fixes

* https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/459

### Description

Enhanced `/agencies` `GET` methods and tests to include linked data sources
LIBTYFI:
* Reorganized and refactored some schemas
* Refactored some SQLAlchemy models
* Refactored `test_select_subquery_relation`
* Refactored some subquery relation logic


### Testing

* Run tests in `tests` directory, confirm all pass as expected
* Inspect API documentation to confirm expected changes to `/agencies` `GET` schemas

### Performance

* Increase of performance overhead in `/agencies` `GET` logic
* Other impacts marginal

### Docs

* API documentation updated

### Breaking Changes

* No breaking changes.
